techniques/flash/FLASH5.html

```diff
@@ -39,13 +39,13 @@ <h3>Using ActionScript to specify the accessible name</h3>
          
             <p>ActionScript 3 can be used instead of the Accessibility control  panel to define the accessibility name for the combined button, as  follows: </p>
          
-         <pre xml:space="preserve">// 'flashLink1' is an instance placed on the movie's main timeline
+         <pre>// 'flashLink1' is an instance placed on the movie's main timeline
```

Comment:
all the naked `<pre> ... </pre>` blocks in this technique could do with inner `<code> ... </code>`